CMakeLists.txt

```diff
@@ -833,8 +852,11 @@ if (ZLIB_ENABLE_TESTS)
     add_executable(switchlevels test/switchlevels.c)
     configure_test_executable(switchlevels)
 
-    add_executable(makefixed tools/makefixed.c inftrees.c)
```

Comment:
The intention here was having this tool always compile as static, since it does not need anything other than two functions from inftrees.c.
I'd also like to add it as a ctest where it runs and compares its output to the fixed table-header we distribute with zlib-ng, I think this would give us good unit test coverage to the two functions in inftrees.c. Anyone feel free to provide a PR adding that, I have not had time to dig into the required ctest stuff yet, and likely will not for a while.

Reply:
I see. I have changed makefixed so it doesn't link against zlib.

Reply:
I think that might be a good idea for a PR. Perhaps there is some place we can list all the TODOs.

Reply:
I have been reviewing alternatives to Trello (wanted to have a kanban board integrated in github), but I didn't find any that met my requirements. 
With Trello, since you need to be a team member to make changes, and any team member pretty much gets admin rights unless you pay for a business account (anyone can delete the whole board for example).

But as far as I can tell, taiga seems to be a better alternative, so I have set up a board that I am experimenting with.
https://tree.taiga.io/project/dead2-zlib-ng-first-stable-release/kanban
I need to figure out how the permissions system works, but I can probably begin inviting people I have the email address of soon(ish), if I dont find something better that is.

Reply:
That looks good. There is also Jira. https://www.atlassian.com/software/views/open-source-license-request

Reply:
I think we need to also add an item for investigating GitHub Actions when it comes out or even Azure Pipelines. Travis CI is awfully slow and Windows support is limited.

Reply:
I use Jira daily at work, and I think it'd be overkill for this project, at least at this point.
If we were to ditch the GitHub issues list, then Jira and others would look a lot better.

What would be great is a KanBan board integrated in the GitHub interface, but I have not found anything like that in the marketplace unfortunately. If it was integrated, then linking to issues and PRs would be easier, and it would be easier to find it.

Btw, buildkite is able to take travis-like configs, I just haven't been able to dedicate the time convert it. It runs on our own hardware, and can run on windows. I have plans to set it up on a couple of Raspberry Pi that I have (a 1 and a 3), so we can run native tests and also do automated benchmarks to spot regressions (requires some scripting).

If only I could spend 100% of my time on this project ;)